src/components/map/Map.js

```diff
@@ -456,12 +460,19 @@ class MapComponent extends Component {
   }
 
   render() {
-    if (this.map && this.state.catFilter && this.state.keyFilter) {
-      this.map.setFilter("pa-tip-projects", [
-        "all",
-        this.state.catFilter,
-        this.state.keyFilter
-      ]);
+    // the filter wasn't working because this.map.setFilter was being called immediately after setting category state,
+    // which is async, and so there was a disconnect between what category state was and what map.setFilter was pulling from
+    // moving setFilter to the render method ensures it will always be filtering the correct state
+    if (this.map) {
+      ["pa-tip-points", "pa-tip-lines"].forEach(layer => {
+        if (this.map && this.state.catFilter && this.state.keyFilter) {
```

Comment:
make sure everything is fully rendered before filtering. Could do:
`if(this.map && this.state.catFilter && this.state.keyFilter) {
    ["pa-tip-points", "pa-tip-lines"].forEach(layer => {
        ...
    })
}
`